wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+  - [Hana - Gekka Jasmine (2014ver.) [Gyro's Light Insane]](https://osu.ppy.sh/beatmaps/1035068)
+
+NB: *Maps were played in the order: NM3, HR2, DT2, HD2, HR1, NM2, DT1, NM1, HD1*
+
+### Qualifying Round 1
+
+**[Download the map pack here! (46.7MB)](https://mega.nz/#!T6RjDY7S!0PbquOystkS4v2-byiBC0b_KMEZA8lECFhMqjYKDm_A)**
+
+- Nomod
+  - [ALiCE'S EMOTiON - Cold Rain [Lunatic]](https://osu.ppy.sh/beatmaps/482650)
+  - [sakuzyo - VALLISTA [Another]](https://osu.ppy.sh/beatmaps/127313)
+  - [U - Ha-tenya? [Insane]](https://osu.ppy.sh/beatmaps/120080)
+  - [YUC'e - Sengoku HOP [apple's Insane]](https://osu.ppy.sh/beatmaps/1489274)
+- Hidden
+  - [Iguchi Yuka - RE-ILLUSION [Collab Insane]](https://osu.ppy.sh/beatmaps/1856697)
+- Hard Rock
+  - [Meramipop - algorhythm [Insane]](https://osu.ppy.sh/beatmaps/136571)
+- Double Time
+  - [T.M.Revolution x Mizuki Nana - Preserved Roses [Collab Hard]](https://osu.ppy.sh/beatmaps/522135)
+
+## Schedules / Match Results
+
+![Schedules](img/schedule_no_number.png)
+
+### Losers Final + Grand Final
+
+Wednesday, 23 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Bubbleman** ![][flag_GB] | **7** | 4 | ![][flag_GB] AJT | [#1](https://osu.ppy.sh/community/matches/55760376) |
+
+Wednesday, 27 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Spare ![][flag_GB] | 3 | **7** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55854629) |
+| Spare ![][flag_GB] | 0 | **7** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55856320) |
+
+### Winners Final + Losers Round 6 / Semi Final
+
+Saturday, 19 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Spare** ![][flag_GB] | **6** | 3 | ![][flag_GB] AJT | [#1](https://osu.ppy.sh/community/matches/55673675) |
+
+Sunday, 20 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Harpy** ![][flag_GB] | **6** | 4 | ![][flag_GB] AJT| [#1](https://osu.ppy.sh/community/matches/55693071) |
+
+Tuesday, 21 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Ventus ![][flag_GB] | 4 | **6** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55719453) |
+
+Monday, 22 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Harpy ![][flag_GB] | 4 | **6** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55738843) |
+
+### Semi Final + Losers Round 4 / 5
+
+Friday, 11 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Malteser ![][flag_GB] | 2 | **6** | ![][flag_GB] **Best459** | [#1](https://osu.ppy.sh/community/matches/55482259) |
+
+Saturday, 12 October 2019
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| bloo ![][flag_GB] | - | **1** | ![][flag_GB] **Kingling** | FF |
```

Comment:
```suggestion
| bloo ![][flag_GB] | -1 | **0** | ![][flag_GB] **Kingling** | *win by default* |
```
I prefer *win by default* (with italics) don't know if it's necessary, plz check clayton.
Scoring is just my example of how i usually takes it.

Reply:
`*win by default*`, `*forfeit*`, anything along these lines is ok. i would avoid abbreviations by themselves cuz ppl might not be familiar with them

Reply:
Went with *forfeit* 👍 leaving the scoring as is, strongly opposed to `-1` as a score